packages/preset-system/src/index.ts

```diff
@@ -67,10 +68,10 @@ export const system = {
     body: 1.5,
     heading: 1.25,
   },
-  textStyles: {
+  text: {
```

Comment:
"textStyles" here was a bug, wasn’t it?

Reply:
I think so 😬

Reply:
I think it was left over from before @theme-ui/components existed; the theme specification on styled-system suggests textStyles as one possible style variant key you might use. I updated this (and some other similar cases) to use `text` as that matches the current usage in components.